src/Common/logger_useful.h

```diff
@@ -54,3 +102,12 @@ namespace
 #define LOG_WARNING(logger, ...) LOG_IMPL(logger, DB::LogsLevel::warning, Poco::Message::PRIO_WARNING, __VA_ARGS__)
 #define LOG_ERROR(logger, ...)   LOG_IMPL(logger, DB::LogsLevel::error, Poco::Message::PRIO_ERROR, __VA_ARGS__)
 #define LOG_FATAL(logger, ...)   LOG_IMPL(logger, DB::LogsLevel::error, Poco::Message::PRIO_FATAL, __VA_ARGS__)
+
+
+#define LOG_TEST_PREFORMATTED(logger, FORMAT_STRING, MESSAGE)    LOG_IMPL_PREFORMATTED(logger, DB::LogsLevel::test, Poco::Message::PRIO_TEST, FORMAT_STRING, MESSAGE)
```

Comment:
Please let's not add another set of logging macros.
1. To the reader, the difference between the existing and the new set of macros isn't immediately clear (this could be resolved with code comments),
2. If someone requires another variant of the log macro in future, he would have to duplicate not 7 but 14 macros, leading to 28 macros in total. These things spread like cancer.
3. There are just two files (out of hundreds) which use the new macros. I am sure that these few places can be converted to the existing macros (maybe at the cost of a negligible performance hit).